dock.pro

```diff
@@ -70,3 +70,7 @@ DISTFILES += \
     dock.json.in \
     version.txt.in \
     README.md
+
+
+# Add setup schema to metadata
+CFG_SCHEMA = "$$cat($$PWD/setup_schema.json)"
```

Comment:
Usually we use a dash instead of an underscore for schema files.
Would be nice to use the same pattern everywhere.

Reply:
I'll update all the integrations!